freqtrade/freqai/prediction_models/PyTorchMLPModel.py

```diff
@@ -0,0 +1,25 @@
+import logging
+
+import torch.nn as nn
+from torch import Tensor
+
+
+logger = logging.getLogger(__name__)
+
+
+class PyTorchMLPModel(nn.Module):
+    def __init__(self, input_dim: int, hidden_dim: int, output_dim: int):
+        super(PyTorchMLPModel, self).__init__()
+        self.input_layer = nn.Linear(input_dim, hidden_dim)
+        self.hidden_layer = nn.Linear(hidden_dim, hidden_dim)
+        self.output_layer = nn.Linear(hidden_dim, output_dim)
+        self.relu = nn.ReLU()
+        self.dropout = nn.Dropout(p=0.2)
```

Comment:
Why not make `p` another kwarg for people to play with? 

Could go even further and parameterize the number of layers using a loop in `forward`. Just a suggestion, you don't need to do it if you prefer not to.

Reply:
yes, that should be added.
i think something like the baselines3 convention should be used here.
freqai: { 
     ...,
     "trainer_kwargs": {
         "max_iters":  100
         "batch_size": 64
         "learning_rate": 3e-4
         "model_kwargs": {
             "n_hidden":  1024,
             "drop_out": 0.2,
        }, 
     },
}

ignoring "model_training_parameters" completely.

Reply:
Using a standard convention is good, but why would we ignore `model_training_parameters`? It fulfills the purpose and it is well known in FreqAI software for serving such purpose. Adding a new dictionary that does the same thing as an existing one is not easy to support.

Reply:
it makes it more clear what are the trainer args and what are the model args.
but i see what you mean, i'll keep `model_training_parameters`.